evap/static/ts/tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "target": "es2019",
         "strict": true,
         "esModuleInterop": true,
-        "moduleResolution": "node"
+        "moduleResolution": "node",
```

Comment:
I think trailing commas are disallowed in standard json - lets revert this line?